### PR TITLE
SC26:carousel with indicator

### PR DESCRIPTION
**Pull request by Shrashi**

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -15,6 +15,8 @@ import Products from "./containers/products/Products";
 import Services from "./containers/services/Services";
 import Contact from "./containers/contact/Contact";
 import SignUp from "./containers/signUp/SignUp";
+import Carousel from "./components/Carousel/Carousel";
+import { carouselItems } from "./utils/constants";
```
Comment (SwarnTyagi) [inline frontend/src/App.js:19]: add spaces in component imports and constant imports

File: frontend/src/components/Carousel/Carousel.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+import { FaAngleDoubleLeft, FaAngleDoubleRight } from "react-icons/fa";
+const ImgStyled = styled.img`
```
Comment (SwarnTyagi) [inline frontend/src/components/Carousel/Carousel.js:4]: add space

Comment (SwarnTyagi) [reply frontend/src/components/Carousel/Carousel.js:4]: in imports and const

File: frontend/src/components/Carousel/Carousel.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  padding: 20px;
+  position: relative;
+`;
+const DivStyled = styled.div`
+  margin-top: 20px;
+  height: 200px;
+  position: relative;
+  padding: 20px;
+`;
+const Carousel = ({ imgConfig, onClickImg }) => {
```
Comment (SwarnTyagi) [inline frontend/src/components/Carousel/Carousel.js:61]: for now, imgConfig is static and we are picking it from constants file
In future we'll be getting it in api

File: frontend/src/components/Carousel/Carousel.js [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+      setOnImg(imgConfig.length - 1);
+    }
+  };
+  const onClickNext = () => {
+    if (onImg >= 0 && onImg < imgConfig.length - 1) {
+      setOnImg(onImg + 1);
+    } else {
+      setOnImg(0);
+    }
+  };
```
Comment (SwarnTyagi) [inline frontend/src/components/Carousel/Carousel.js:82]: try, if we can combine the logic in single function with some func params

File: frontend/src/components/Carousel/Carousel.js [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+  const onClickImage = ({ src, alt, imageNo, url }) => {
+    onClickImg && onClickImg(src, alt, imageNo, url);
```
Comment (SwarnTyagi) [inline frontend/src/components/Carousel/Carousel.js:84]: pass it object only, don't pass multiple params

File: frontend/src/components/Carousel/Carousel.js [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+    </DivStyled>
+  );
+};
+
+const CarouselIndicator = ({ config, imageNo }) => {
+  return (
+    <ListStyled>
+      {config.map((ele) => {
+        const selected = ele.imageNo === imageNo ? "primary" : "secondary";
+
```
Comment (SwarnTyagi) [inline frontend/src/components/Carousel/Carousel.js:108]: kill extra spaces if not needed

File: frontend/src/utils/constants.js [hunk 7]
```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import img1 from "../assets/carousel-item1.jpg";
```
Comment (SwarnTyagi) [inline frontend/src/utils/constants.js:2]: space please

**Review by SwarnTyagi — CHANGES_REQUESTED** [review]
Please do the needful

File: frontend/src/components/Carousel/Carousel.js (hunk 3 above)
Comment (Shrashi) [reply frontend/src/components/Carousel/Carousel.js:61]: Okay.

File: frontend/src/components/Carousel/Carousel.js (hunk 4 above)
Comment (Shrashi) [reply frontend/src/components/Carousel/Carousel.js:82]: Done

**Review by SwarnTyagi — APPROVED** [review]

**Pull request merged**